README.md

```diff
@@ -8,3 +8,61 @@ It includes Javascript and native stack traces, heap statistics,
 platform information and resource usage etc. With the report enabled,
 reports are triggered on unhandled exceptions, fatal errors, signals
 and calls to a Javascript API.
+
+Usage:
+
+    npm install nodejs/nodereport
+
+    var nodereport = require('nodereport');
+
+By default, this will trigger a NodeReport to be written to the current
```

Comment:
I don't think this is a good idea. It means that there is no way to opt-out of auto-handling of `unhandledException`, which would be useful for someone who wanted to only use the manual `.triggerReport()`. I think it would make more sense for a require of `nodereport` to make no changes to global process state until it was attached. Syntax could be `require('nodereport')()`, or `require('nodereport').attach()`, or any other variant.